### PR TITLE
Add sidecard component

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- Add sidecard component

## Screenshots

sidecard1
sidecard2

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- local
- storybook

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/sidecard/Sidecard.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <h2 class="bcgov-color-heading">{{ heading }}</h2>
+        <div class="bcgov-submit-content">
+          <span>{{ content }}</span>
+        </div>
+      </section>
+      <section v-if="type === 'blue'" id="blue-section" class="bcgov-submit-container">
+        <h2 class="bcgov-heading-style">{{ heading }}</h2>
+        <div class="bcgov-submit-content">
+          <span>{{ content }}</span>
+          <a v-if="image && imageLink" v-bind:href="imageLink" target="_blank" rel="noopener noreferrer">
```
Comment (karthikmsqa) [inline src/components/sidecard/Sidecard.vue:19]: nit: noopener noreferrer => should these be 'no-opener' or camel case?

File: src/components/sidecard/Sidecard.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    padding-right: 10px;
+  }
+
+  .bcgov-wide-dashboard-spacing {
+    padding-left: 0;
+  }
+}
+
+.bcgov-side-card-row {
+  white-space: nowrap;
```
Comment (karthikmsqa) [inline src/components/sidecard/Sidecard.css:109]: nit: 'nowrap' => should it be 'no-wrap' or camel case?

File: src/components/sidecard/Sidecard.vue (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/components/sidecard/Sidecard.vue:19]: The rel=”noreferrer” tag is a special HTML attribute that can be added to a link tag (<a>). It prevents passing the referrer information to the target website by removing the referral info from the HTTP header.

so it has to be in the casing currently used as it is a preset html attribute

File: src/components/sidecard/Sidecard.css (hunk 2 above)
Comment (sdevalapurkar-bcgov) [reply src/components/sidecard/Sidecard.css:109]: Screen Shot 2020-08-31 at 2 54 26 PM

Comment (sdevalapurkar-bcgov) [reply src/components/sidecard/Sidecard.css:109]: once again preset

Comment (karthikmsqa) [reply src/components/sidecard/Sidecard.css:109]: Thank  you. Learnt something!

File: src/components/sidecard/Sidecard.vue (hunk 1 above)
Comment (karthikmsqa) [reply src/components/sidecard/Sidecard.vue:19]: Thank you Shreyas!

**Pull request merged**